test/unit/Middleware.t.sol

```diff
@@ -1484,10 +1487,30 @@ contract MiddlewareTest is Test {
 
         vm.store(address(middleware), slot, bytes32(0));
 
+        vm.roll(80);
+
         vm.expectRevert(IMiddleware.Middleware__GatewayNotSet.selector);
         middleware.sendCurrentOperatorsKeys();
     }
 
+    function testSendCurrentOperatorKeysButIsLimitedByMaxIntervalBlock() public {
```

Comment:
Would also change name from Max to Min. Because that is the minimum number of blocks you need to wait.